lib/bolt/config.rb

```diff
@@ -159,7 +170,7 @@ def update_from_cli(options)
 
       TRANSPORT_OPTIONS.each do |key|
         TRANSPORTS.each do |transport|
-          unless %i[ssl host_key_check].any? { |k| k == key }
+          unless %i[ssl host_key_check task_environment].any? { |k| k == key }
```

Comment:
do we accept task environment from the cli?

Reply:
Not that I could find.